cli/test/test_cmddot.py

```diff
@@ -0,0 +1,43 @@
+from click.testing import CliRunner
+
+import popper.commands.cmd_dot as dot
+import popper.commands.cmd_run as run
+from test_common import PopperTest
+from popper.parser import Workflow, YMLWorkflow, HCLWorkflow
+
+import unittest
+import os
+import tempfile
+import git
+
+
+class TestDot(PopperTest):
+
+    def test_dot(self):
+
+        wf_dir = tempfile.mkdtemp()
+        runner = CliRunner()
+        git.Git(wf_dir).clone(
+            "https://github.com/popperized/github-actions-demo.git")
+
+        workflow_loc = f'{wf_dir}/github-actions-demo/.github/main.workflow'
```

Comment:
May we can change the name to `workflow_path`.

Reply:
Yes sure